src/hyperion/parameters/schemas/full_external_parameters_schema.json

```diff
@@ -3,7 +3,7 @@
     "type": "object",
     "properties": {
         "params_version": {
-            "const": "4.0.2"
+            "const": "4.0.3"
```

Comment:
Must: We will need a corresponding GDA change that updates to have this as the expected param version and changes all hyperion calls to use `expected_energy` too